docs/pages/deployment/pex.rst

```diff
@@ -0,0 +1,97 @@
+.. _pex:
+
+Presentation Definition mapping
+###############################
+
+Wallet functionality uses `Presentation Definitions <https://identity.foundation/presentation-exchange/>`_ to determine the required credentials for a given presentation request.
+An OAuth2 authorization request uses scopes to determine the required permissions for a given request.
+The mapping between scopes and presentation definitions is defined in a configuration file or by a policy backend.
+
+Configuration
+*************
+
+The Nuts config supports two ways to define the mapping between OAuth2 scopes and presentation definitions:
+- using a file-based configuration
+- using a policy backend
+
+The file-based configuration is a simple way to define the mapping between scopes and presentation definitions.
+It can be used for simple use cases where the mapping is static and does not change often.
+
+To use file-based configuration, you need to define the path to a directory that contains policy configuration files:
+
+.. code-block:: yaml
+
+    policy:
+        directory: /path/to/directory
```

Comment:
Since https://github.com/nuts-foundation/nuts-node/commit/c9fe559f905562463efe53679f6e6fd627b5ef5b there's a default directory, so we might not need to document this in detail here.